Splitted two booking flows to separate services

mingazhev opened the pull request:
Credit card and account flows differ much and do not have real duplicate code so I've splitted them to separate small services.
+ in addition I've removed capturing credit card payment if deadline passed - we've authorized money before and I don't think that it is vital to capture at the same time.
This is discussable

kirillta commented on Api/Services/Accommodations/Bookings/Flows/ICreditCardBookingFlow.cs:
```diff
@@ -0,0 +1,14 @@
+
+namespace HappyTravel.Edo.Api.Services.Accommodations.Bookings.Flows
+{
+    public interface ICreditCardBookingFlow
```
Both class names are ambiguous. What about `IBankCardBookingFlow` and `IFinancialAccountBookingFlow`?

mingazhev replied:
Renamed

kirillta commented:
> * in addition I've removed capturing credit card payment if deadline passed - we've authorized money before and I don't think that it is vital to capture at the same time

That's OK while one cannot cancel such bookings

mingazhev commented:
> 
> 
> > * in addition I've removed capturing credit card payment if deadline passed - we've authorized money before and I don't think that it is vital to capture at the same time
> 
> That's OK while one cannot cancel such bookings

Good note, reverted money capture, at least for now
